pymc3/variational/callbacks.py

```diff
@@ -11,24 +11,52 @@ def __call__(self, approx, loss, i):
         raise NotImplementedError
 
 
+def percentage(current, prev, eps=1e-6):
+    return (np.abs(current - prev)+eps)/(np.abs(prev)+eps)
+
+
+def absolute(current, prev):
+    return np.abs(current - prev)
+
+_diff = dict(
+    percentage=percentage,
+    absolute=absolute
+)
+
+
 class CheckParametersConvergence(Callback):
-    def __init__(self, every=1000, tolerance=1e-3, eps=1e-10):
+    """
+
+    Parameters
+    ----------
+    every : int
+        check frequency
+    tolerance : float
+        if diff norm < tolerance : break 
+    diff : str
+        difference type one of {'absolute', 'percentage'}
```

Comment:
Shouldn't this be called `relative` instead of "percentage"? Its not multiplied by 0.01. Also, I can't remember seeing that specified as a percentage before.